docs(components): add mutations annotation docs (#785)
GenSpectrum/dashboard-components: components/src/web-components/MutationAnnotations.mdx

import { Meta } from '@storybook/blocks';

<Meta title='Concepts/Mutation Annotations' />

# Mutation Annotations

> The organism data usually have a lot of mutations.

Reviewer: ```suggestion
Organism data usually has a lot of mutations.
```
